admin_guide/limits.adoc

```diff
@@ -87,7 +87,36 @@ specified.
 <13> The default amount of memory that a container will request to use if not specified.
 <14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
 ====
-// end::admin_limits_sample_definition[]
+
+.OpenShift Limit Range Object Definition
+====
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "openshift-resource-limits"
+spec:
+  limits:
+    -
```

Comment:
Take a look at the [generated page](https://miminar.fedorapeople.org/openshift-docs/image-limits/admin_guide/limits.html#limit-range-def) with concise notation.

![Image concise notation](https://miminar.fedorapeople.org/openshift-docs/concise-notation.png)


    $ asciibinder -v
    0.1.3.2

I'll open an issue. Until fixed, I'd rather keep the expanded notation.